std/path.d

```diff
@@ -1080,14 +1080,14 @@ auto setExt(R, C)(R path, C[] ext)
 unittest
 {
     import std.array;
-    assert (setExt("file", "ext").array == "file.ext");
-    assert (setExt("file"w, ".ext"w).array == "file.ext");
-    assert (setExt("file.ext"w, ".").array == "file.");
+    assert (withExtension("file", "ext").array == "file.ext");
+    assert (withExtension("file"w, ".ext"w).array == "file.ext");
+    assert (withExtension("file.ext"w, ".").array == "file.");
 
     import std.utf : byChar, byWchar;
-    assert (setExt("file".byChar, "ext").array == "file.ext");
-    assert (setExt("file"w.byWchar, ".ext"w).array == "file.ext"w);
-    assert (setExt("file.ext"w.byWchar, ".").array == "file."w);
+    assert (withExtension("file".byChar, "ext").array == "file.ext");
+    assert (withExtension("file"w.byWchar, ".ext"w).array == "file.ext"w);
+    assert (withExtension("file.ext"w.byWchar, ".").array == "file."w);
```

Comment:
This example (documented unittest) would look much better if UFCS was used, should I update it?